packages/components/src/unit-control/test/index.tsx

```diff
@@ -373,18 +373,20 @@ describe( 'UnitControl', () => {
 			const units = [
 				{ value: 'pt', label: 'pt', default: 0 },
 				{ value: 'vmax', label: 'vmax', default: 10 },
+				{ value: '+', label: '+', default: 10 },
```

Comment:
```suggestion
				// Proves that units with regex control characters don't error.
				{ value: '+', label: '+', default: 10 },
```
I like the idea of noting the significance of this but I don't think anyone’s going to haphazardly change the test on us 🤷.